.travis.yml

```diff
@@ -0,0 +1 @@
+script: bash -ex scripts/build_website.sh
```

Comment:
I'm pretty sure we need more here, but I could be wrong. Like specifying that we need python and such.